lib/song_page.dart

```diff
@@ -337,123 +329,127 @@ class _MySongPageState extends State<MySongPage> {
       ));
     }
 
-    return Scaffold(
-      // @see https://api.flutter.dev/flutter/widgets/NestedScrollView-class.html
-      body: SafeArea(
-        child: OrientationBuilder(builder: (context, orientation) {
-          return Container(
-            // TODO Make themeable.
-            color: Colors.white,
-            // Some physical devices (like Xiaomi Mi A2) display a one-pixel
-            // high white line above the song title for some weird reason. Let's
-            // hide that with this very small top margin.
-            margin: const EdgeInsets.fromLTRB(0, 0.01, 0, 0),
-            child: Flex(
-              direction: orientation == Orientation.portrait
-                  ? Axis.vertical
-                  : Axis.horizontal,
-              children: [
-                Expanded(
-                  child: NestedScrollView(
-                    controller: scrollController,
-                    headerSliverBuilder: ((context, innerBoxIsScrolled) {
-                      return [
-                        SliverOverlapAbsorber(
-                          handle:
-                              NestedScrollView.sliverOverlapAbsorberHandleFor(
-                                  context),
-                          sliver: SliverAppBar(
-                            pinned: orientation == Orientation.portrait,
-                            // @see https://github.com/flutter/flutter/issues/79077#issuecomment-1226882532
-                            expandedHeight: 57,
-                            title: Text(
-                              getSongTitle(widget.songsInBook[songKey]),
-                              style: const TextStyle(fontSize: 18),
-                              maxLines: 2,
-                            ),
+    return Consumer<SettingsProvider>(builder: (context, provider, child) {
+      return Scaffold(
+        // @see https://api.flutter.dev/flutter/widgets/NestedScrollView-class.html
+        body: OrientationBuilder(builder: (context, orientation) {
+          return Flex(
+            direction: orientation == Orientation.portrait
+                ? Axis.vertical
+                : Axis.horizontal,
+            children: [
+              Expanded(
+                child: NestedScrollView(
+                  controller: scrollController,
+                  headerSliverBuilder: ((context, innerBoxIsScrolled) {
+                    return [
+                      SliverOverlapAbsorber(
+                        handle: NestedScrollView.sliverOverlapAbsorberHandleFor(
+                            context),
+                        sliver: SliverAppBar(
+                          pinned: orientation == Orientation.portrait,
+                          // @see https://github.com/flutter/flutter/issues/79077#issuecomment-1226882532
+                          expandedHeight: 57,
```

Comment:
A görgetési problémát nem tudom reprodukálni.
Én is pont ugyanezt tapasztaltam a NestedScrollView betételekor, de ezzel a sorral elvileg meg kéne oldva legyen.
@boobaa kérlek próbáld ki az adott eszközre rábuldelni úgy, hogy növeled ezt az értéket még párral, fellép-e a gond.
Viszont annyi gond van ezzel a NesterScrollView-al, hajlok rá hogy kivegyük és fekvő módban az appbart egyszerűen betenném a Column-ba, ami görög.

Reply:
> növeled ezt az értéket még párral, fellép-e a gond
57, 58, 59, 60, 65, -100 (sic!), 56, 55, 54, 53 értékekkel nem láttam semmilyen különbséget, a gond ugyanúgy fennáll. 100-as értéknél a csík ugyanúgy megjelenik, cserébe viszont az elütő színű fejléc is magasabb (lehet, hogy már 65-nél is, csak nem tűnt fel).

> annyi gond van ezzel a NesterScrollView-al, hajlok rá hogy kivegyük és fekvő módban az appbart egyszerűen betenném a Column-ba, ami görög
Egy próbát megér.

Reply:
Tisztázva. Kell az expandedHeight.